frontend/app/routes/protected/renew/$id/confirm-home-address.tsx

```diff
@@ -0,0 +1,260 @@
+    country: String(formData.get('homeCountry')),
+    province: formData.get('homeProvince') ? String(formData.get('homeProvince')) : undefined,
+    city: String(formData.get('homeCity')),
+    postalCode: formData.get('homePostalCode') ? String(formData.get('homePostalCode')) : undefined,
+  });
+
+  if (!parsedDataResult.success) {
+    return data({ errors: transformFlattenedError(parsedDataResult.error.flatten()) }, { status: 400 });
+  }
+
+  saveProtectedRenewState({ params, session, state: { homeAddress: parsedDataResult.data } });
+
+  return redirect(getPathById('protected/renew/$id/review-adult-information', params));
+}
+
+export default function ProtectedRenewConfirmHomeAddress() {
+  const { t } = useTranslation(handle.i18nNamespaces);
+  const { defaultState, countryList, regionList } = useLoaderData<typeof loader>();
+  const { CANADA_COUNTRY_ID, USA_COUNTRY_ID } = useClientEnv();
+  const params = useParams();
+  const fetcher = useFetcher<typeof action>();
+  const isSubmitting = fetcher.state !== 'idle';
+  const [selectedHomeCountry, setSelectedHomeCountry] = useState(defaultState.homeAddress?.country ?? CANADA_COUNTRY_ID);
+  const [homeCountryRegions, setHomeCountryRegions] = useState<typeof regionList>([]);
+
+  const errors = fetcher.data?.errors;
+  const errorSummary = useErrorSummary(errors, {
+    address: 'home-address',
+    province: 'home-province',
+    country: 'home-country',
+    city: 'home-city',
+    postalCode: 'home-postal-code',
+  });
+
+  const countries = useMemo<InputOptionProps[]>(() => {
+    return countryList.map(({ id, name }) => ({ children: name, value: id }));
+  }, [countryList]);
+
+  useEffect(() => {
+    const filteredProvinceTerritoryStates = regionList.filter(({ countryId }) => countryId === selectedHomeCountry);
+    setHomeCountryRegions(filteredProvinceTerritoryStates);
+  }, [selectedHomeCountry, regionList]);
+
+  const homeCountryChangeHandler = (event: React.SyntheticEvent<HTMLSelectElement>) => {
+    setSelectedHomeCountry(event.currentTarget.value);
+  };
+
+  // populate home region/province/state list with selected country or current address country
+  const homeRegions: InputOptionProps[] = homeCountryRegions.map(({ id, name }) => ({ children: name, value: id }));
+
+  const dummyOption: InputOptionProps = { children: t('protected-renew:update-address.address-field.select-one'), value: '' };
+
+  const postalCodeRequiredContries = [CANADA_COUNTRY_ID, USA_COUNTRY_ID];
+  const homePostalCodeRequired = postalCodeRequiredContries.includes(selectedHomeCountry);
+
+  return (
+    <div className="max-w-prose">
+      <p className="mb-4 italic">{t('renew:required-label')}</p>
+      <errorSummary.ErrorSummary />
+      <fetcher.Form method="post" noValidate>
+        <CsrfTokenInput />
+        <fieldset className="mb-8">
+          <legend className="mb-4 font-lato text-2xl font-bold">{t('protected-renew:update-address.home-address.header')}</legend>
```

Comment:
This extra header can be removed (it no longer exists in Figma). We can also remove the `<fieldset>` as a result of removing this `<legend>`.

(same thing for mailing address page)